shared/middleware/helpers/storage_id.rb

```diff
@@ -89,13 +89,17 @@ def storage_id_cookie_name
   name
 end
 
-def storage_id_for_user
+def storage_id_for_user_id(user_id)
+  row = user_storage_ids_table.where(user_id: user_id).first
+  row[:id] if row
+end
```

Comment:
Need to be able to get a storage_id for a specific user for the case where a teacher is viewing a student's solution.